### PR TITLE
Auto-minify JS and CSS.

### LINKED ISSUE #218: Minify the JS we serve
low priority, but would be nice to reduce the size a little

imo we should add a comment at the top with a link to the unminified source, in the spirit of open source

### PR DESCRIPTION
**Pull request by redplanks**
Should resolve #218 

Unfortunately, yui-compressor's output wildcard syntax fails when there's only one input file through the wildcard, so I had to hardcode minifying the CSS file.

### CODE REVIEW

**Comment by jvperrin** [comment]
The CSS one worked fine, but the JS one didn't seem to work, I'll try it again though

**Comment by jvperrin** [comment]
Also, does this minify the [vhost_mail.js](https://static.ocf.berkeley.edu/js/pages/account/vhost_mail.js) file as well?

**Comment by redplanks** [comment]
Did a bit of script appropriation, so now both CSS and JS are recursively minified.

File: Dockerfile.static.in [hunk 1]
```diff
@@ -1,6 +1,8 @@
 FROM {tag}
 
 RUN /opt/ocfweb/venv/bin/sassc /opt/ocfweb/ocfweb/static/scss/site.scss /opt/ocfweb/ocfweb/static/scss/site.scss.css
+RUN /opt/ocfweb/recursive-minify css /opt/ocfweb/ocfweb/
+RUN /opt/ocfweb/recursive-minify js /opt/ocfweb/ocfweb/
```
Comment (chriskuehl) [inline Dockerfile.static.in]: i'd be tempted to drop the script and just write (untested, but probably works)

```dockerfile
RUN find \( -name '*.js' -o -name '*.css' \) -exec yui-compressor -o {} {} \;
```

/shrug

Comment (redplanks) [reply Dockerfile.static.in]: I tested it and it does work. Thank you for the suggestion!

**Pull request merged**